js/modules/offline-exporting.src.js

```diff
@@ -6,11 +6,54 @@
  * License: www.highcharts.com/license
  */
 
+/**
+ * Download options for offline exporting.
+ *
+ * @typedef Highcharts.OfflineExportingDownloadOptionsObject
+ *
+ * @property {string} [filename]
+ *           Name of resulting downloaded file without extension.
+ *
+ * @property {string} [libURL]
+ *           URL pointing to location of dependency scripts to download on
+ *           demand. By default, the module will load these file from our
+ *           server. Internet Explorer requires the canvg library in order to
+ *           export to PNG and to export charts with embedded images. PDF export
+ *           also requires the jsPDF and svg2pdf for all browsers.
+ *
+ * @property {number} [scale]
+ *           Scaling factor of downloaded image compared to source
+ *
+ * @property {string} [type]
+ *           File type of resulting download
+ */
+
+/**
+ * @typedef Highcharts.OfflineExportingOptionsObject
```

Comment:
I believe these are the same options that are available for the online exporting, so they should probably use the same typedef.